qulice-checkstyle/src/test/resources/com/qulice/checkstyle/LocalVariableNames.java

```diff
@@ -7,6 +7,7 @@
  * Simple.
  * @version $Id$
  * @author John Smith (john@example.com)
+ * @checkstyle FinalLocalVariable (51 lines)
```

Comment:
@mkordas it`s just to stop final variable restriction at this test case. I think this case is just to test variables names, correct? Can we leave this like it is?